actiontext/app/helpers/action_text/content_helper.rb

```diff
@@ -22,17 +22,32 @@ def render_action_text_attachments(content)
       content.render_attachments do |attachment|
         unless attachment.in?(content.gallery_attachments)
           attachment.node.tap do |node|
-            node.inner_html = render(attachment, in_gallery: false).chomp
+            node.inner_html = render_action_text_attachment attachment, locals: { in_gallery: false }
           end
         end
       end.render_attachment_galleries do |attachment_gallery|
         render(layout: attachment_gallery, object: attachment_gallery) do
           attachment_gallery.attachments.map do |attachment|
-            attachment.node.inner_html = render(attachment, in_gallery: true).chomp
+            attachment.node.inner_html = render_action_text_attachment attachment, locals: { in_gallery: true }
             attachment.to_html
           end.join.html_safe
         end.chomp
       end
     end
+
+    #:nodoc:
+    def render_action_text_attachment(attachment, locals: {})
```

Comment:
```suggestion
    def render_action_text_attachment(attachment, locals: {}) #:nodoc:
```

Otherwise all methods below that one will also be nodoc.

Reply:
Done!